magento/pwa-studio
# Bugfix/transparent png  #1547

## Description

Update to latest version of express-sharp v2.1.2

## Verification Steps
- Upload .png image to magento backend product
- Go to product page.
- Check if image has transparent background

Closes #1547.
